crates/task-impls/src/consensus.rs

```diff
@@ -1034,7 +1034,17 @@ where
                     self.update_view(view + 1).await;
                 }
             }
+            SequencingHotShotEvent::VidCertRecv(cert) => {
+                debug!("VID cert received for view ! {}", *cert.view_number);
 
+                let view = cert.view_number;
+                self.certs.insert(view, cert);
```

Comment:
Currently `self.certs` stores the DAC type.  We should probably update it to store both the VIDCert type and DAC type.

Reply:
new issue https://github.com/EspressoSystems/HotShot/issues/1701